assemblies/pentaho-cde-renderer/karaf/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.pentaho.ctools</groupId>
+    <artifactId>pentaho-cde-renderer-assembly</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pentaho-cde-renderer-karaf-assembly</artifactId>
+
+  <packaging>karaf-assembly</packaging>
+
+  <properties>
+    <karaf.version>3.0.8</karaf.version>
+    <karaf-maven-plugin.version>4.1.4</karaf-maven-plugin.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <!-- scope is compile so all features (there is only one) are installed into startup.properties and the feature repo itself is not added in etc/org.apache.karaf.features.cfg file -->
+      <groupId>org.apache.karaf.features</groupId>
+      <artifactId>framework</artifactId>
+      <version>${karaf.version}</version>
+      <type>kar</type>
+    </dependency>
+    <dependency>
+      <!-- scope is runtime so the feature repo is listed in etc/org.apache.karaf.features.cfg file, and features will installed into the system directory -->
+      <groupId>org.apache.karaf.features</groupId>
+      <artifactId>standard</artifactId>
+      <version>${karaf.version}</version>
+      <classifier>features</classifier>
+      <type>xml</type>
+      <scope>runtime</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.pentaho.webpackage</groupId>
+      <artifactId>pentaho-webpackage</artifactId>
+      <version>${pentaho.version}</version>
```

Comment:
Use the proper version property name: https://github.com/pentaho/build-resources/blob/8.1.0.0/version.properties#L101